src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/UseConcreteTypeTests.cs

```diff
@@ -0,0 +1,614 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Performance.UseConcreteTypeAnalyzer,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
+
+namespace Microsoft.NetCore.Analyzers.Performance.UnitTests
+{
+    public static partial class UseConcreteTypeTests
+    {
+        [Fact]
+        public static async Task Params()
+        {
+            const string Source = @"
+                namespace Example
+                {
+                    public interface IFoo
+                    {
+                        void Bar();
+                    }
+
+                    public class Foo : IFoo
+                    {
+                        public void Bar() {}
+                    }
+
+                    public class Test
+                    {
+                        private void Method(IFoo {|#0:foo|})
```

Comment:
It would be better to squiggle below the symbol that needs an action, i.e. it would make more sense to squiggle under  `IFoo` not `foo` ( `{|#0:IFoo|} foo` )